tinygrad/helpers.py

```diff
@@ -353,3 +353,5 @@ def dfs(x:Any, cache:dict):
   if (cx:=cache.setdefault(x, [0,0,False]))[2]: return f"{' '*d} x{cx[0]}"
   cx[2], srcs = True, ('None' if srcfn(x) is None else ''.join(f'\n{pretty_print(s, rep, srcfn, cache, d+2)},' for s in srcfn(x)))
   return f"{' '*d}{f'x{cx[0]}:=' * (cx[1]>1)}{rep(x)}" % srcs
+
+ReductionStr = Literal["mean", "sum", "none"]
```

Comment:
Why is this in helpers? Is it used anywhere besides Tensor?

Reply:
nah, and probably never will be... I moved it to tensor.py

Reply:
Also rebased and squashed